addons/mail/static/src/less/composer.less

```diff
@@ -8,7 +8,80 @@
         margin-right: 30px;
     }
 }
+.o_modal_fullscreen {
+    padding: 0;
+    .o_viewer_content {
+        width: 100%;
+        height: 100%;
+        min-height: 250px;
+        position: relative;
+        cursor: pointer;
+        .o_viewer-header {
+            color: #FFFFFF;
+            position: absolute;
```

Comment:
Can't see why we would need to absolute-position every elements... this is probably wrong.